webapp/components/team_general_tab.jsx

```diff
@@ -131,7 +131,7 @@ class GeneralTab extends React.Component {
 
         var data = this.props.team;
         data.allow_open_invite = this.state.allow_open_invite;
-        Client.updateTeam(data,
+        updateTeam(data,
             (team) => {
                 TeamStore.saveTeam(team);
                 TeamStore.emitChange();
```

Comment:
Sorry, one more minor request. Since you've added storing the team in the action, can you remove the

```
TeamStore.saveTeam(team);
TeamStore.emitChange();
```
from here and from the other two uses of `updateTeam` in here? It'll be redundant now